app/mailers/user_mailer.rb

```diff
@@ -46,17 +44,82 @@ def work_collaborator(user, work)
+
+    def initialize(user:, added_works:, active_pages:, active_translations:, active_note_pages:)
+      @user = user
+      @added_works = added_works
+      @active_pages = active_pages
+      @active_translations = active_translations
+      @active_note_pages = active_note_pages
+    end
+
+    class << self
+      def build(user)
+        #find which pages the user has worked on
+        user_page_ids ||= user.deeds.pluck(:page_id).uniq.compact
+
+        Activity.new(
+          {
+            user: user,
+            added_works: works_added_to_users_collection_in_past_day(user),
+            active_pages: user_pages_edited_in_past_day(user, user_page_ids),
+            active_translations: user_pages_translated_in_past_day(user, user_page_ids),
+            active_note_pages: user_pages_with_notes_added_in_past_day(user, user_page_ids)
+          }
+        )
+      end
+
+      private
+
+      def user_pages_edited_in_past_day(user, user_page_ids)
+        recently_modified_pages = Page.joins(:deeds).where(deeds: {deed_type: DeedType.edited_and_transcribed_pages}).merge(Deed.past_day).distinct
+        #find pages that have been newly edited by someone other than the user (the user is not the last editor)
+        recently_modified_pages.where(id: user_page_ids).select {|page| page if page.deeds.where(deed_type: DeedType.edited_and_transcribed_pages).last.user_id != user.id}
+      end
+
+      def user_pages_translated_in_past_day(user, user_page_ids)
+        recently_edited_translation_pages = Page.joins(:deeds).where(deeds: {deed_type: DeedType.new_and_edited_translations}).merge(Deed.past_day).distinct
+        # find translation pages that have been newly edited by someone other than the user
+        recently_edited_translation_pages.where(id: user_page_ids).select {|page| page if page.deeds.where(deed_type: DeedType.new_and_edited_translations).last.user_id != user.id}
+      end
+
+      def user_pages_with_notes_added_in_past_day(user, user_page_ids)
+        pages_with_recent_notes = Page.joins(:deeds).where(deeds: {deed_type: DeedType::NOTE_ADDED}).merge(Deed.past_day).distinct
+        # find pages that the user has worked on that has had notes added recently
+        pages_with_recent_notes.where(id: user_page_ids).select {|page| page if page.deeds.where(deed_type: DeedType::NOTE_ADDED).last.user_id != user.id}
+      end
+
+      def works_added_to_users_collection_in_past_day(user)
+        # collections the user has worked in
+        user_collection_ids = user.deeds.pluck(:collection_id).uniq
+        # works that have been added to those collections by someone other than the user in the past day
+        Work.where(collection_id: user_collection_ids).joins(:deeds).where(deeds: {deed_type: DeedType::WORK_ADDED}).merge(Deed.past_day).where.not(deeds: {user_id: user.id}).distinct
+      end
+    end #end class << self
+
+    def has_contributions?
+      (
+        @added_works +
```

Comment:
@benwbrum I noticed that there were queries in place to get `works_added_to_users_collection_in_past_day`, which we were capturing in `added_works`, but we weren't actually using that to determine if a work had contributions. This may have been a bug. I added `added_works` into that logic, which will result in emails being sent when a work is added.